lib/jsdom/browser/new-resource-loader.js

```diff
@@ -0,0 +1,131 @@
+"use strict";
+
+const fs = require("fs");
+const { parseURL } = require("whatwg-url");
+const { fromURLRecord } = require("data-urls");
```

Comment:
Maybe rename this while importing so it's clearer, e.g. `const dataURLFromRecord = require("data-urls").fromURLRecord`.

Reply:
Do you prefer that way, or `const { fromURLRecord: dataURLFromRecord } = require("data-urls");` is ok?